tests/machine_learning_utilities_test/conftest.py

```diff
@@ -0,0 +1,333 @@
+"""
+Test configuration for machine learning utilities.
+
+This conftest.py provides fixtures and configuration specific to pressure models
+located in <project root>/tests/machine_learning_utilities_test/
+"""
+
+import contextlib
+import os
+import tempfile
+from unittest.mock import patch
+
+import numpy as np
+import pytest
+
+# Import all model classes for fixtures - adjusted for test directory structure
+from rock_physics_open.equinor_utilities.machine_learning_utilities import (
+    ExponentialPressureModel,
+    FriableDryBulkModulusPressureModel,
+    FriableDryShearModulusPressureModel,
+    PatchyCementDryBulkModulusPressureModel,
+    PatchyCementDryShearModulusPressureModel,
+    PolynomialPressureModel,
+    SigmoidalPressureModel,
+)
+
+
+# Test-specific pytest configuration
+def pytest_configure(config):
+    """Configure markers specific to machine learning utilities tests."""
+    config.addinivalue_line("markers", "unit: marks tests as unit tests")
+    config.addinivalue_line("markers", "integration: marks tests as integration tests")
+    config.addinivalue_line(
+        "markers", "pressure_model: marks tests related to pressure models"
+    )
+    config.addinivalue_line("markers", "slow: marks tests as slow running")
+    config.addinivalue_line("markers", "benchmark: marks performance benchmark tests")
+
+
+def pytest_collection_modifyitems(config, items):
```

Comment:
Suggestion: Add type annotations here:

```suggestion
def pytest_collection_modifyitems(config: pytest.Config, items: Iterable[pytest.Item]):
```